lib/framework.js

```diff
@@ -462,7 +466,10 @@ class Framework {
 			config.beforeMiddleware.push("ui5--pauseRequests");
 			config.middleware.push("ui5--serveResources");
 			// config.middleware.push("ui5--serveThemes");
-			server = await this.setupUI5Server(config.basePath);
+			server = await this.setupUI5Server({
+				basePath: config.basePath,
+				configPath: config.ui5.configPath
```

Comment:
could `ui5` be `undefined` if not used in the configuration => causing an undefined exception?

Reply:
Then it would have failed in line 462 already...

Reply:
OK - wasn't visible in the review

Reply:
even line 465 wasn't visible to me in the review... 😄